doc/api/deprecations.md

```diff
@@ -2498,6 +2498,39 @@ Type: Runtime
 Passing a callback to [`worker.terminate()`][] is deprecated. Use the returned
 `Promise` instead, or a listener to the worker’s `'exit'` event.
 
+<a id="DEP00XX"></a>
+### DEP00XX: Closing fs.FileHandle on Garbage Collection
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/28396
+    description: Runtime deprecation.
+-->
+
+Type: Runtime
+
+Allowing a [`fs.FileHandle`][] object to be closed on garbage collection is
+deprecated. In the future, doing so will result in a thrown error that will
```

Comment:
Optional nit: Since we can't be 100% sure about the future, perhaps `s/will/may/`?